packages/vite-plugin-checker/src/codeFrame.ts

```diff
@@ -1,30 +1,39 @@
 import os from 'node:os'
-import type ts from 'typescript'
 
 import { type SourceLocation, codeFrameColumns } from '@babel/code-frame'
 
-export function createFrame({
-  source,
-  location,
-}: {
-  source: string // file source code
-  location: SourceLocation
-}) {
-  const frame = codeFrameColumns(source, location, {
-    highlightCode: true,
+/**
+ * Create a code frame from source code and location
+ * @param source source code
+ * @param location babel compatible location to highlight
+ */
+export function createFrame(source: string, location: SourceLocation): string {
+  return codeFrameColumns(source, location, {
+    // worker tty did not fork parent process stdout, let's make a workaround
+    forceColor: true,
   })
     .split('\n')
     .map((line) => `  ${line}`)
     .join(os.EOL)
-
-  return frame
 }
 
-export function tsLocationToBabelLocation(
-  tsLoc: Record<'start' | 'end', ts.LineAndCharacter /** 0-based */>
+export function tsLikeLocToBabelLoc(
+  tsLoc: Record<'start' | 'end', { line: number; character: number } /** 0-based */>
 ): SourceLocation {
   return {
     start: { line: tsLoc.start.line + 1, column: tsLoc.start.character + 1 },
     end: { line: tsLoc.end.line + 1, column: tsLoc.end.character + 1 },
   }
 }
+
+export function locationToBabelLocation(d: {
```

Comment:
i was thinking about renaming it to something like, `lineColLocToBabelLocation` or maybe something like this,

```
normalizeTsLikeLoc
normalizeLineColLoc
```

Reply:
`lineColLocToBabelLoc` is good enough. 👍